### PR TITLE
feat: add is_current_filter for offers view

### PR DESCRIPTION
**Pull request by long74100**
# ⛔️ DEPRECATION WARNING

**This repository is deprecated and in maintainence-only operation while we work on a replacement, please see [this announcement](https://discuss.openedx.org/t/deprecation-removal-ecommerce-service-depr-22/6839) for more information.**

Although we have stopped integrating new contributions, we always appreciate security disclosures and patches sent to [security@edx.org](mailto:security@edx.org)

## Required Testing
- [ ] Before deploying this change, complete a purchase in the stage environment. 
(^ We can remove that manual check once REV-2624 is done and the corresponding e2e test runs again)

## Description

Describe what this pull request changes, and why these changes were made. How will these changes affect other people, installations of edx, etc.?
Please include links to any relevant ADRs, design artifacts, and decision documents. Make sure to document the rationale behind significant changes in the repo, per [OEP-19](https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html), and can be
linked here.

Useful information to include:
- Which edX user roles will this change impact? Common user roles are "Learner", "Course Author", "Developer", and "Operator".
- Include screenshots for changes to the UI (ideally, both "before" and "after" screenshots, if applicable).
- Provide links to the description of corresponding configuration changes. Remember to correctly annotate these changes.

## Supporting information

Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
Be sure to check they are publicly readable, or if not, repeat the information here.

## Testing instructions

Please provide detailed step-by-step instructions for testing this change; how did YOU test this change?

## Other information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, OpenEdx vs. edx.org differences, development vs. production environment differences, security, or accessibility.

### CODE REVIEW

File: ecommerce/extensions/api/v2/tests/views/test_enterprise.py [hunk 1]
```diff
@@ -4140,7 +4156,9 @@ def test_admin_view_is_current(self, start_datetime, end_datetime, expected_is_c
             kwargs={'enterprise_customer': enterprise_customer_uuid},
         )
         response_json = self.client.get(path).json()
-        assert response_json['results'][0]['is_current'] == expected_is_current
+        assert response_json['results'][0]['display_name'] == expected_display_name
+
+
```
Comment (christopappas) [inline ecommerce/extensions/api/v2/tests/views/test_enterprise.py:4220]: nit: lots of new lines

File: ecommerce/extensions/offer/tests/test_models.py [hunk 2]
```diff
@@ -552,6 +556,56 @@ def test_creating_offer_with_null_partner(self):
...
+            'expected_is_current': False,
+        },
+    )
+    @ddt.unpack
+    def test_is_current(self, start_datetime, end_datetime, expected_is_current):
+        offer = factories.ConditionalOfferFactory(
+            start_datetime=start_datetime,
+            end_datetime=end_datetime,
+        )
+        assert offer.is_current == expected_is_current
```
Comment (christopappas) [inline ecommerce/extensions/offer/tests/test_models.py]: nit: asserting for bools should use `is`
```
assert offer.is_current is expected_is_current
```

**Review by christopappas — APPROVED** [review]
makes sense to me. just a tiny nit

**Review by adamstankiewicz — APPROVED** [review]

File: ecommerce/extensions/api/v2/views/enterprise.py [hunk 3]
```diff
@@ -188,10 +188,12 @@ def get_queryset(self):
                 attribute_values__value_boolean=False
             )
 
+            now = timezone.now()
             queryset = queryset.filter(code__in=Product.objects
                                        .filter(product_class__name=COUPON_PRODUCT_CLASS_NAME)
                                        .filter(active_coupon)
-                                       .filter(coupon_vouchers__vouchers__end_datetime__gte=now())
+                                       .filter(coupon_vouchers__vouchers__start_datetime__lte=now)
```
Comment (adamstankiewicz) [inline ecommerce/extensions/api/v2/views/enterprise.py:195]: Good catch

**Review by adamstankiewicz — APPROVED** [review]

**Pull request merged**